set.mm

```diff
@@ -77653,21 +77653,25 @@ any sets (which usually are functions) and any element (even not
     EFUPZSVGETUQWCVKVIRMRWCVJRVIWCAFGZVCHQVJRNWCVCWEWDSAETUQPVIURUSUTVAVB $.
 
   ${
-    $d x F $.  $d x G $.  $d x Z $.  $d x U $.  $d x W $.  $d x A $.  $d x B $.
```

Comment:
If something is not important in itself, why not mark it as a lemma of other theorems?  I can see that you were the author of this theorem, so, ok, I am willing to accept this deletion.  In general, though, there is a tendency to stay stable, even if a theorem loses during a development phase its references.

Maybe suppcoss is a sort of lemma as well?

Reply:
yeah, in retrospect suppcofnd should've been marked as a lemma.

I could see suppcoss being marked as a lemma too, though note that I also used it in mplsubglem (which goes similarly to mhpinvcl for the invcl part of subg). Not sure if that changes anything

Reply:
The idea here is that a lemma is tied to a target theorem.  If the proof of this theorem changes, or if it gets deleted altogether, then the lemmas will vanish too.  If you instead have a wider spread application range, i.e. at least two unrelated applications, then it is a more general utility that you want to keep even if one or both applications are dismissed.  I don't know what idea is prevailing.  That's your choice.

Reply:
I'd say it's useful enough not to be a lemma, then. It feels similarly as useful as suppofssd